docs/smart_scripts.md

```diff
@@ -263,7 +263,7 @@ This is the probability of the event to occur as a percentage from 0-100. So, if
-| SMART_ACTION_DIE                                | 37    |                                                                                                                                                                                                                                                                                                                                                                                                                                        |                                                                                                      |                                                                                                                            |                                                |                                       |                                       | Kill Target                                                                                                                                                                                                                                                                                                                                 |
+| SMART_ACTION_DIE                                | 37    | Milliseconds                                                                                                                                                                                                                                                                                                                                                                                                                          |                                                                                                      |                                                                                                                            |                                                |                                       |                                       | Kill Target, if param1 is set, it will die after param1 milliseconds. 0: instakill                                                                                                                                                                                                                                                                      |
```

Comment:
```suggestion
| SMART_ACTION_DIE                                | 37    | Milliseconds                                                                                                                                                                                                                                                                                                                                                                                                                          |                                                                                                      |                                                                                                                            |                                                |                                       |                                       | Kill Target, if param1 is set, it will die after param1 milliseconds. 0: no delay, the target will die instantly                                                                                                                                                                                                                                                                      |
```